Gemfile

```diff
@@ -61,6 +61,7 @@ gem 'will_paginate', '>= 3.0.6'
 gem 'will_paginate-bootstrap', '>= 1.0.1'
 gem 'georuby', '2.0'
 gem 'geokit-rails'
+gem 'json'
```

Comment:
Hi, @StlMaris123 - sorry, i still see this Gemfile change. Can you try one more time to remove this change and we can merge it?